### ISSUE TITLE
Add `root#description` configuration option

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor5#20119]
## 📝 Provide a description of the improvement

We have now `root#label`, but it's meant for screen readers and general a11y, so it won't always be as descriptive as it could/should be.

## 📃 Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.
